src/main/java/school/lemon/changerequest/java/generics/container/ContainerImpl.java

```diff
@@ -0,0 +1,69 @@
+package school.lemon.changerequest.java.generics.container;
+
+/**
+ * Created by Yaroslav Pavlinskiy on 26.12.2016.
+ */
+public class ContainerImpl<E> implements Container<E>{
+    private int size;
+    private E[] array;
+
+    ContainerImpl()
+    {
+        this.array = (E[]) new Object[10];
```

Comment:
I'm not sure that such class cast will work. You cannot cast `Object` to Type `E[]`.

Please provide Demo class for Container to check whether your implementation work correctly.
Provide demo for all operations for some 2 different Generic type, for example: `String` and `Float`.